backend/compact-connect/tests/unit/test_app.py

```diff
@@ -76,6 +76,29 @@ def test_synth_sandbox(self):
 
         self._inspect_api_stack(app.sandbox_stage.api_stack)
 
+    def test_synth_sandbox_no_domain(self):
+        """
+        Test infrastructure as deployed in a developer's sandbox
```

Comment:
Can you add a brief addendum to this comment? Like...

..."which does not set a domain (falling back to XX)"